apps/web/playwright/booking-pages.test.ts

```diff
@@ -58,32 +57,24 @@ test.describe("free user", () => {
     await expect(page.locator("[data-testid=booking-fail]")).toBeVisible();
   });
 
-  // Why do we need this test. The previous test is testing /30min booking only ?
-  todo("`/free/30min` is bookable");
-
-  test("`/free/60min` is not bookable", async ({ page }) => {
+  test("Second event type is not bookable", async ({ page, users }) => {
+    const [free] = users.get();
     // Not available in listing
-    await expect(page.locator('[href="/free/60min"]')).toHaveCount(0);
+    await expect(page.locator(`[href="/${free.username}/${free.eventTypes[1].slug}"]`)).toHaveCount(0);
 
-    await page.goto("/free/60min");
+    await page.goto(`/${free.username}/${free.eventTypes[1].slug}`);
     // Not available on a direct visit to event type page
     await expect(page.locator('[data-testid="404-page"]')).toBeVisible();
   });
 });
 
 test.describe("pro user", () => {
-  test.use({ storageState: "playwright/artifacts/proStorageState.json" });
-
-  test.beforeAll(async () => {
-    await deleteAllBookingsByEmail("pro@example.com");
-  });
-
-  test.beforeEach(async ({ page }) => {
-    await page.goto("/pro");
+  test.beforeEach(async ({ page, users }) => {
+    const pro = await users.create();
+    await page.goto(`/${pro.username}`);
   });
-
-  test.afterAll(async () => {
-    await deleteAllBookingsByEmail("pro@example.com");
+  test.afterEach(async ({ users }) => {
+    await users.deleteAll();
```

Comment:
Isn't this like the one above? It doesn't conflict with anything still being run?

Reply:
The users fixture only contains the users that we're created by that specific worker for that specific test. So it's only deleting all users that were created for this test.

Reply:
Got it!